### ISSUE TITLE
Use the cache with jsonapi_render.

### ISSUE THREAD

**Issue by GCorbel** [tiagopog/jsonapi-utils#84]
I noticed than this doesn't use the cache : 

```
  def index
    jsonapi_render json: ProjectGroup.all
  end
```

But this does : 

```
  def index
    process_request
  end
```

Is there a way to use the cache for both? Should I cache records myself? It can be useful to have this "built-in".

**Comment by tiagopog** [comment]
Yes, initially the idea was to let the cache implementation to be a developer's choice. But I will reconsider this point since caching usually is not related to the app's business logic.

Thanks for the feedback :-)
